<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="3.33s"
		durationms="3330"
		type="root"
		rid="1bd4f55100358338"
		totalerrorcount="0"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="I-Ray"
		rxversion="10.5.1.0"
		host="IRAYSERVER"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="en-US"
		dtnlanguage="en-US"
		screenresolution="1600x900"
		timestamp="11/15/2023 2:15:19 PM"
		timestampiso="2023-11-15T08:45:19"
		endtime="11/15/2023 2:15:22 PM"
		timeoutfactor="1"
		runid=""
		sutversion="">
		<detail>
			Ranorex Testscript
		</detail>
		<item
			timeRelativeToTestSuiteStartTime="00:03.318"
			timeRelativeToTestModuleStartTime="00:03.318"
			timeWallClock="14:15:22"
			time="00:03.318"
			level="Failure"
			category="TestSuite">
			<message>
				Thread was being aborted.
			</message>
			<metainfo
				stacktrace="   at Microsoft.Win32.NativeMethods.ShellExecuteEx(ShellExecuteInfo info)
   at System.Diagnostics.ShellExecuteHelper.ShellExecuteFunction()
   at System.Diagnostics.ShellExecuteHelper.ShellExecuteOnSTAThread()
   at System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at Ranorex.Core.Controls.TestProgressProvider.StartTestProgressProvider(ProviderParams providerParams)
   at Ranorex.Core.Testing.TestSuiteRunner.RunInternal(Type containerType, String[] testSuiteXmls, RunParams[] rps)"
				loglvl="Failure">
			</metainfo>
		</item>
	</activity>
</report>
